src/routes/device-type/add.ts

```diff
@@ -37,7 +52,22 @@ const controllerGeneratorOptions: ControllerGeneratorOptions = {
       pinFieldConfigurations: fieldConfigurationsToServerSchema.required()
         .description('Defines deviceFields on the pin the channel is connected to. Can be used in report type functions. See the chapter on open fields on how to use this'),
       defaultPinName: Joi.string().example('Anode').description('If undefined, the channel cannot be linked to a pin'),
+      charts: Joi.array().items(Joi.object().keys({
+        title: Joi.string().allow(null).example('Red wire charts').required(),
+        series: Joi.array().items(Joi.object().keys({
+          quantityHashId: Joi.string().example('x18a92').required(),
+          color: Joi.string().example('#ff00ff').required(),
+        })),
```

Comment:
Line 60 should get a `.required()` I think, to be in line with the typescript Interface (which is correct).

Reply:
oh, fixed )